operators/tokenizer/bpe_json.hpp

```diff
@@ -9,6 +9,8 @@
 
 #include "bpe_types.h"
 
+#include <filesystem>
```

Comment:
filesystem is not available for all platforms that ort-extensions needs to support, try to use some C API.